visualization/app/codeCharta/ui/fileChooser/fileChooser.component.spec.ts

```diff
@@ -163,4 +164,5 @@ describe("fileChooserController", () => {
 			expect(fileChooserController["files"][1].content.fileChecksum).toBe("a35dd32bd850841c47894ac9b8e1287a")
 		})
 	})
+	describe("", () => {})
```

Comment:
Remove empty test

Reply:
Oh, slipped under the radar!
Corrected in the next commit.